tilequeue/command.py

```diff
@@ -2309,6 +2311,7 @@ def tilequeue_meta_tile_low_zoom(cfg, args):
         coords.extend(coord_children_range(parent, group_by_zoom - 1))
 
     for coord in coords:
+        coord_start_ms = int(time.time() * 1000)
```

Comment:
just curious why you're making this into millis - in all places, you don't have access to the milli info, you're just multiplying by 1000, so the precision is not real;  For things under 1 second, you're going to see either 0ms or 1000ms.  Maybe just be ok with second granularity?

Reply:
> just curious why you're making this into millis - in all places, you don't have access to the milli info, you're just multiplying by 1000, so the precision is not real; For things under 1 second, you're going to see either 0ms or 1000ms. Maybe just be ok with second granularity?

Graphite has precision to milliseconds and time.time() returns a float where it has some precision to the milliseconds level.

Reply:
ahhh.... that's weird that Time.time returns a float, good ole python